test/test_csv2cve.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+from os.path import dirname, join
+
+import pytest
+
+import cve_bin_tool.csv2cve as csv2cve
+from cve_bin_tool.error_handler import ERROR_CODES, InsufficientArgs
+
+
+class TestCSC2CVE:
+    @pytest.mark.asyncio
+    async def test_csv2cve_main(self, caplog):
```

Comment:
Nitpick
I'm guessing we are testing csv2cve with a valid csv file and one with an invalid file. IMO it might be better for the test to have explicit names like `test_csv2cve_valid_csv` and `test_csv2cve_invalid_file`

Reply:
Alright!